packages/fonts/src/font-weights.ts

```diff
@@ -1,52 +1,40 @@
 export const fontWeights = {
   "100": {
     label: "Thin",
-    name: "thin",
-    alt: "hairline",
+    names: ["thin", "hairline"],
   },
   "200": {
     label: "Extra Light",
-    name: "extra light",
-    alt: "ultra light",
+    names: ["extra light", "extralight", "ultra light", "ultralight"],
```

Comment:
I was trying to normalize these, why would you add this stuff that is essentially all the same?

Reply:
ah you added all possible names to just be able to match in the end with the number?

Reply:
Not quite. Some fonts have subfamily with spaces, some without spaces. Here I explicitly listed all.

Reply:
yes